src/Commands/RetryBulkRequestCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace JustBetter\MagentoAsync\Commands;
+
+use Illuminate\Console\Command;
+use JustBetter\MagentoAsync\Contracts\RetriesBulkRequest;
+use JustBetter\MagentoAsync\Models\BulkRequest;
+
+class RetryBulkRequestCommand extends Command
+{
+    protected $signature = 'magento:async:retry-bulk-request {id} {--only-failed}';
```

Comment:
In most cases, the `--only-failed` is preferable. Perhaps rename it to `--all` which would by `false` without any options? This way, you could only retry the failed transactions by default.

Reply:
Then I'd prefer to default it to true (which the Nova implementation does too), the action isn't called `RetryFailed`. This is just an optional filter.

Reply:
Fair! In that case, it can remain what it is